webapp/locales/de.json

```diff
@@ -829,7 +829,10 @@
   },
   "user": {
     "avatar": {
-      "submitted": "Erfolgreich hochgeladen!"
+      "submitted": "Avatar-Bild erfolgreich hochgeladen!"
+    },
+    "profileHeader": {
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-06-19T14:30:26Z" title="Friday, June 19th 2020, 4:30:26 pm +02:00">Jun 19, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -830,6 +830,9 @@
   "user": {
     "avatar": {
       "submitted": "Erfolgreich hochgeladen!"
+    },
+    "profileHeader": {
+      "submitted": "Erfolgreich hochgeladen!"
     }
```


May we can use one identifier for both cases if we show the same text?
like this:
```json
"image": {
  "submitted": "Erfolgreich hochgeladen!“
}
```

Or we do different texts as example:
- `Avatar-Bild erfolgreich hochgeladen!`
- `Profilbild erfolgreich hochgeladen!`

Reply:
> [<img alt="DriesCruyskens" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/DriesCruyskens) **Authored by [DriesCruyskens](https://github.com/DriesCruyskens)**
_<time datetime="2020-06-23T15:42:14Z" title="Tuesday, June 23rd 2020, 5:42:14 pm +02:00">Jun 23, 2020</time>_

---


I'll do the different texts one since it is better to letter the user know exactly what has changed to eliminate any confusion.